README.md

```diff
@@ -24,15 +24,21 @@ all the required libraries pre-installed.
 make run generate
 ```
 
-### Build the Docker image
+### Build the Container image
 
 After regenerating the gRPC gateway stubs, rebuild the `rest-proxy` Docker image.
 
 ```bash
 make build
 ```
 
-### Push the Docker image
+If you need to use another builder than `docker`, you can specify it by using the `BUILDER` parameter:
+
+```bash
+make BUILDER=podman build
```

Comment:
could this style of Makefile parameter throw off commands like `make run fmt`? If so maybe we go with shell env variables instead? i.e. `ENGINE=podman make build`

Reply:
It works as well:

```
BUILDER=podman make run fmt
Makefile:79: warning: overriding commands for target `fmt'                                                                                                                                                                                              ─╯
Makefile:63: warning: ignoring old commands for target `fmt'
podman build -t kserve/rest-proxy-develop:latest --target develop .
time="2023-11-30T15:19:59-03:00" level=warning msg="missing \"BUILDOS\" build argument. Try

...

Successfully tagged localhost/kserve/rest-proxy-develop:latest
0d8582b2343843c7cc7c91033d525e60b548bf78cac9a84ce6c61bab895d61b7
BUILDER=podman && ./scripts/develop.sh make fmt
./scripts/fmt.sh
golangci-lint............................................................Passed
prettier.................................................................Passed
```